README.md

```diff
@@ -28,30 +32,59 @@
 
 Configuration Anomaly Detection (CAD) is responsible for reducing manual SRE effort by pre-investigating alerts, detecting cluster anomalies and sending relevant communications to the cluster owner.
 
-## Contributing
+## Contributing 
 
-To contribute to CAD, please see our [CONTRIBUTING Document](CONTRIBUTING.md).
+### Adding a new investigation
 
-# Documentation
+CAD investigations are triggered by PagerDuty webhooks. Currently, CAD supports the following two formats of webhooks:
+-  WebhookV3 
+-  EventOrchestrationWebhook
 
-## CAD CLI
+The required investigation is identified by CAD based on the incident and its payload. 
+As PagerDuty itself does not provide finer granularity for webhooks than service-based, CAD filters out the alerts it should investigate. For more information, please refer to https://support.pagerduty.com/docs/webhooks.
+
+To add a new alert investigation:
+- create a mapping for the alert to the `getInvestigation` function in `investigate.go` and write a corresponding CAD investigation (e.g. `Investigate()` in `chgm.go`).
+- if the alert is not yet routed to CAD, add a webhook to the service your alert fires on.
```

Comment:
are we covered in terms of side effects when adding this webhook?

15 min auto escalate on cad failures for example

Reply:
Quickly checked against our services, the services escalation policy needs to be updated as well to "Openshift Escalation Policy Automation (Stage)"

Reply:
I updated it and kept it broad as this depends on what we want to do: for staging per example, we don't really care about the escalation policy when it comes to just running `cadctl`.